### ISSUE TITLE
Millerview Preview Problem

### ISSUE THREAD

**Issue by TheMystifyingCharacter** [vifm/vifm#706]
I seem to have the same problem that is explained in [this issue](https://github.com/vifm/vifm/issues/656). It still doesn't seem to work as expected. If I have an image preview and scroll to a directory, the image stays. I am testing this with `vifm-git` from the AUR and the latest version of `ueberzug` with my own preview script. In case it was my fault, I also tested the latest version of `vifmimg`. The preview can still be cleared by doing `!vifmimg clear` or by entering or leaving a directory.
I have
```
set previewoptions+=graphicsdelay:0
set milleroptions=lsize:1,csize:2,rsize:3,rpreview:all
set millerview
```
in my `vifmrc` if it helps. And I copied the preview part exactly from vifmimg's README.

**Comment by xaizek** [comment]
That issue was about changing directory rather than moving cursor and I didn't notice the issue when you go from a graphical file to a directory. Thanks for the report!

**Comment by TheMystifyingCharacter** [comment]
It's no problem at all. Thanks for maintaining such a useful file manager!

**Comment by TheMystifyingCharacter** [comment]
I figured I might as well add this. I also found another problem, I think. It also involves millerview. With the same settings as above, if I have true colors for vifm (I set `$TERM` to `xterm-direct` and `kitty-direct`), the background of previews (text and graphical but not for directories) will be black. For me, using kitty (also tried Alacritty), it will be color0 no matter what highlight options are set (I tested it by turning everything into one color). It doesn't matter if you have gui* or cterm* set, unset, or anything. There will be a black (or whatever color you have it set to) rectangle. I'm wondering if it's a problem with me changing `$TERM` or not. I had to change it because I know kitty has true color support but vifm doesn't detect it when `$TERM` is `xterm-kitty` and `:help` also mentioned using `-direct`.
